src/main/java/uk/co/ramp/covid/simulation/output/CsvOutput.java

```diff
@@ -0,0 +1,61 @@
+package uk.co.ramp.covid.simulation.output;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CsvOutput {
+    private static final Logger LOGGER = LogManager.getLogger(CsvOutput.class);
+
+    public static void writeDailyStats(Appendable out, int startIterID, List<List<DailyStats>> stats) throws IOException {
```

Comment:
Maybe in the future we might think about making these non-static, with the class having a reference to the outDir/DailyStats etc. Then we can have a single "writeOutput" method call in model that let's us extend exactly what the output does without adding extra calls (if we want, for example, a carehome deaths file or whatever).